demos/apps/sleep/sleep.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: v1
```

Comment:
I thought the source svc thing was still an open discussion. If not, I wonder if there's any need to have this sample service at all. Maybe just deploying busybox, like you did it in your integrating services task, is sufficient for all the things we may want to experiment with.

I'm not quite sure what to do now, so I think I'm just going to merge this as it is for now, so I can finish the egress task using it, and we can revisit whether or not to keep, remove, or change it in the future.